README.md

```diff
@@ -320,6 +320,31 @@ class MyLeapGame extends LeapGame {
 }
 ```
 
+## Debugging
+
+### Slow motion
+
+`LeapWorld` includes
+[`HasTimeScale`](https://pub.dev/documentation/flame/latest/components/HasTimeScale-mixin.html),
+so you can set `world.timeScale = 0.5` to slow your whole game down to 50% speed to make it easier
+to play test nuanced bugs. (You can use this as slow motion for your game too.)
+
+### Render hitbox
+
+`PhysicalEntity` includes a `debugHitbox` property you can override which will automatically
+draw a box indicating the exact hitbox the collision detection system is using for your
+entity.
+
+```dart
+class Player extends PhysicalEntity {
+
+  /// Draw entity's hitbox
+  @override
+  bool get debugHitbox => true;
```

Comment:
I wonder if it wouldn't be better if this was just a field instead, then the developer would not need to override this but rather just set it to true.